tests/test_span.py

```diff
@@ -232,3 +232,12 @@ def test_span_component(tracer):
 
     span.set_tag(ext_tags.COMPONENT, 'crypt')
     assert span.component == 'crypt'
+
+
+def test_span_tag_value_max_length(tracer):
+    tracer.max_tag_value_length = 42
+    span = tracer.start_span(operation_name='x')
+    span.set_tag('x', 'x' * 50)
```

Comment:
Could you add a test with unicode as well?

Reply:
String truncation works fine with unicode. I don't see a point of testing a feature of the language.

```
>>> x=u'key2-caf\xe9-blah'
>>> print(x)
key2-café-blah
>>> print(x[:10])
key2-café-
>>> print(x[:9])
key2-café
>>> print(x[:8])
key2-caf
```